Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs

```diff
@@ -1,12 +1,13 @@
-﻿using Ecommerce.ViewModels.PaymentDetail;
+﻿using Ecommerce.Application.ViewModels.PaymentDetail;
+using Ecommerce.Domain.Entities;
 
-namespace Ecommerce.Mappings
+namespace Ecommerce.Application.Mappings
 {
```

Comment:
![image](https://github.com/user-attachments/assets/95769dd0-23dc-42fd-a623-47e748a7a624)
Convert file scoped namespace

Reply:
I wonder if the screenshot is from another project
sorry but i dont understand :(

Reply:
Yes it's a different project and it has a compiler error.
A semicolon should be at the end of the namespace line;

Reply:
I understand now